dlf/common/class.tx_dlf_helper.php

```diff
@@ -845,17 +845,7 @@ public static function getURN($base, $id) {
 	 */
 	public static function intInRange($theInt, $min, $max = 2000000000, $zeroValue = 0) {
 
-		if (t3lib_utility_VersionNumber::convertVersionNumberToInteger(TYPO3_version) >= 6000000) {
-
-			// TYPO3 > 6.0
-			return t3lib_utility_Math::forceIntegerInRange($theInt, $min, $max, $zeroValue);
-
-		} else {
-
-			// TYPO3 4.5 - 4.7
-			return \TYPO3\CMS\Core\Utility\GeneralUtility::intInRange($theInt, $min, $max, $zeroValue);
-
-		}
+		return t3lib_utility_Math::forceIntegerInRange($theInt, $min, $max, $zeroValue);
```

Comment:
Maybe we should completely remove this helper function and replace all calls to it with a direct call to `t3lib_utility_Math::forceIntegerInRange`? The helper function was only introduced to have a downwards compatibility for TYPO3 4.5, which is no longer needed.